caesar_cipher/lib/caesar_cipher.rb

```diff
@@ -0,0 +1,33 @@
+class CaesarCipher
+  @@alphabet_length = 'z'.ord - 'a'.ord + 1
+
+  def initialize(num)
+    @num = num
+  end
+
+  def encrypt(str)
+    str
+      .split('')
+      .map do |l|
+        progressed_letter(l)
+      end
+      .join('')
+  end
+
+  def progressed_letter(l)
+    if l.match(/[a-z]/)
```

Comment:
If, for performance, we didn't want to use regex here, we could have done something like `between?('a'.ord, 'z'.ord)`